azure-ts-aks-keda/README.md

```diff
@@ -0,0 +1,96 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Azure Kubernetes Service (AKS) cluster and Azure Functions with KEDA
+
+This example demonstrates creating an Azure Kubernetes Service (AKS) Cluster, and deploying an Azure Function App with Kubernetes-based Event Driven Autoscaling (KEDA) into it, all in one Pulumi program. Please see https://docs.microsoft.com/en-us/azure/aks/ for more information about AKS and https://docs.microsoft.com/en-us/azure/azure-functions/functions-kubernetes-keda for more information about KEDA.
+
+# Prerequisites
+
+Ensure you have [downloaded and installed the Pulumi CLI](https://pulumi.io/install).
+
+We will be deploying to Azure, so you will need an Azure account. If you don't have an account,
+[sign up for free here](https://azure.microsoft.com/en-us/free/).
+[Follow the instructions here](https://pulumi.io/install/azure.html) to connect Pulumi to your Azure account.
+
+This example deploys a Helm Chart from Kedacore Helm chart repository, so you
+will need to [install the Helm CLI](https://docs.helm.sh/using_helm/#installing-helm) and configure it:
+
+```bash
+$ helm init --client-only
+$ helm repo add kedacore https://kedacore.azureedge.net/helm
+```
```

Comment:
You will also need to run `helm update` after the repo is added to make sure everything is up to date.

Reply:
[Helm example](https://github.com/pulumi/examples/tree/master/azure-ts-aks-helm) doesn't mention this. Should I add in both?

Reply:
Yes, please. See https://github.com/kedacore/keda#setup